libs/components/ag-grid/src/lib/modules/ag-grid/ag-grid.service.ts

```diff
@@ -662,7 +663,21 @@ export class SkyAgGridService implements OnDestroy {
     return defaultSkyGridOptions;
   }
 
-  #onCellFocused(): void {
+  #onCellFocused(event: CellFocusedEvent): void {
+    if (!event.column || typeof event.column !== 'object') {
+      return;
+    }
+    const rowNode =
+      Number.isInteger(event.rowIndex) &&
+      event.api.getDisplayedRowAtIndex(event.rowIndex as number);
+    if (!rowNode) {
+      return;
+    }
+
+    if (event.column.isRowDrag(rowNode) || event.column.isDndSource(rowNode)) {
+      return;
```

Comment:
What do you think about putting these `if` conditions in a `#isDragging()` method (or something similar) so that the code is quickly understood when we debug this in the future?
```
if (this.#isDragging(event)) {
  return;
}
```

Reply:
Good call. Updated.